thoth/storages/graph/postgres.py

```diff
@@ -799,6 +799,46 @@ def _construct_solved_python_packages_query(
         kwargs.pop("self", None)  # static method
         return self.__class__._construct_python_packages_query(**kwargs)
 
+    def get_solver_document_id_for_package_version(
```

Comment:
Shouldn't be just another filter for: https://github.com/thoth-station/storages/blob/b29d1606d5dce208e18012e456900810293284ed/thoth/storages/graph/postgres.py#L4323

Reply:
+1, good catch. Will move them there.